lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
       --><button type="submit" class="button postfix discovery-submit" aria-label="Search">
```

Comment:
I'm not crazy about the approach taken for this button.  There is no accessible text here except for the aria-label.  Aria-label should only be used as a last resort.  It's not universally supported so it may not work in all environments. Ideally, we would have an SVG icon in here.  SVG supports accessible text so at least there would be something between the ```<button></button>``` elements.  Right now, we only have an icon font (which may not load, especially if the user has customized there browser fonts) and the "Loading" text which is not the primary label for this control.  I would prefer offscreen text here for the text value of this button.